microros_esp32_extensions/main/Kconfig.projbuild

```diff
@@ -53,4 +53,86 @@ menu "WiFi Configuration"
             Set the Maximum retry to avoid station reconnecting to the AP unlimited when the AP is really inexistent.
 endmenu
 
+endmenu
+
+
+	menu "Camera configuration"
```

Comment:
Is there a possibility that we can set this as an application-specific configuration?

Reply:
Related to: https://github.com/micro-ROS/micro_ros_setup/pull/176#discussion_r514341827

Maybe we can just explain how to configure this in the app README.md of create a config.h that includes all this headers. Are they system-wide?

Reply:
I have opened a PR for this [here](https://github.com/espressif/esp32-camera/pull/194). if approved the full camera configurion will be inside the menuconfig only when esp32-camera  repository is in the components directory (`esp-idf/components`).